### PR TITLE
fix(a11y): update element roles and aria expanded to be set if false

### PR DESCRIPTION
**Pull request by tobias-g**
Hi

We noticed some aria issues and role issues our [aXe](https://github.com/dequelabs/axe-core) tests came up with.

`aria-expanded={menuOpen}` that was on the input when it was false because of the binding setup the attribute was not added at all when expected behaviour is when false the attribute is present and displayed as `aria-expanded="false"`.

Also moved the aria-expanded attribute and combobox role to the wrapper elements, then added role textbox to the input as aXe was not happy with this either.

Hope that all makes sense cheers 🍺

### CODE REVIEW

File: test/functional/index.js [hunk 1]
```diff
@@ -68,6 +68,26 @@ describe('Autocomplete', () => {
         expect(scratch.innerHTML).to.contain('class="bob__input')
         expect(scratch.innerHTML).to.contain('class="bob__menu')
       })
+
+      it('renders with the correct aria attributes', () => {
+        render(<Autocomplete required />, scratch)
+
+        console.log(scratch)
```
Comment (tvararu) [inline test/functional/index.js]: You forgot a `console.log` here. 😄

Comment (tobias-g) [inline test/functional/index.js]: Ha indeed 😱 my bad, fixed that now 👍

**Comment by tobias-g** [comment]
@tvararu I pushed updates to the branch to remove my console.log. Anything else to update drop me another message ta :)

**Comment by danmitchell-** [comment]
@tobias-g if you can rebase with master then I can merge this for you 👍

**Comment by tobias-g** [comment]
Thanks @danmitchell- rebase done 😁

**Pull request merged**